src/Driver.hpp

```diff
@@ -63,6 +61,79 @@ namespace dvl_teledyne
          * Throws std::runtime_error if an error is reported by the device
          */
         void readConfigurationAck(base::Time const& timeout = base::Time::fromSeconds(1));
+
+        /** Configures the number of bottom-track pings per data ensemble.*/
+        void setBottomTrackPingsPerEnsemble(int bottomTrackPingsPerEnsemble);
+
+        /** Configures the maximum tracking depth (10 - 65535 dm).*/
+        void setMaximumTrackingDepth(float maximumTrackingDepth);
+
+        /** Configures the serial port control settings (stopBits : 1 or 2).*/
+        void setSerialPortControlSettings(BAUDRATE baudrate, PARITY parity, int stopBits);
+
+        /** Configures wether ensemble and ping cycling are handled
+         * automatically or manually, wether data output is sent in binary or
+         * hex-ASCII format, and wether data is sent via the serial interface or not.
+         */
+        void setFlowControlSettings(bool automaticEnsembleCycling,
+                                    bool automaticPingCycling,
+                                    bool binaryDataOutput,
+                                    bool enableSerialOutput,
+                                    bool enableDataRecording);
+
+        /** Configures the correction between the heading reference and beam 3 of the
+         * DVL (-17999 - 18000 hundreths of a degree).
```

Comment:
The documentation is ambiguous ... one could think it applies to the function's argument.
```suggestion
         * DVL
         * @param headingAlignment offset in radians, between -179.99 and +180 degrees
```

Maybe use a `base::Angle` instead of a plain float ?

Reply:
Thanks forgot to update the documentation. I also adapted the rest of the documentation to this style where it seemed appropriate.

Reply:
Using base::Angle seems like a good idea though the documentation states it only holds values
 PI < value <= PI
so I think there is a negative sign missing.